.github/ISSUE_TEMPLATE/issue_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Issue report
+about: Create a report to help us improve MetaMorpheus
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describing the issue**
+What issue did you encounter?
+
+**Reproducing the error**
+Which protein database did you use? What type of mass spectrometry data files did you use? If possible, please provide examples of the datafiles that reproduce the error. 
+
+What settings did you use? Did you use the defaults? 
+
+**Expected behavior**
+If the issue has to do with interacting with the GUI, what did you expected to happen instead?
+
+**Screenshots**
+If applicable, pleas add screenshots to help explain the problem.
+
+**Desktop information**
+ - OS: [e.g. Windows 10]
+ - MetaMorpheus Version: [e.g. 0.0.320]
+ - Available RAM: [e.g. 32 GB]
```

Comment:
This information is contained in the Autogenerated manuscript prose file:

![image](https://user-images.githubusercontent.com/16841846/143085661-922816ca-879c-472f-ae27-50c04820c950.png)

Reply:
Noted that they can find it in the prose file!